### ISSUE TITLE
oracle_database_tables.sql in  openid-connect-server-webapp project

### ISSUE THREAD

**Issue by excaliburKJ** [mitreid-connect/OpenID-Connect-Java-Spring-Server#1317]
CREATE TABLE client_details (
...
...
software_statement VARCHAR(4096),
  software_id VARCHAR(2048),
  software_statement VARCHAR2(4000),
...
...
}
should be "software_version"?

**Comment by angelok1** [comment]
Yes! I have a pull request out there with this corrected. Any chance we can get it merged?
